jetson-nano.coffee

```diff
@@ -0,0 +1,44 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BOARD_POWEROFF = 'Remove power from the board and insert frelshy burnt SD-CARD.'
+BOARD_POWERON  = 'Power on the board.'
+
+module.exports =
+	version: 1
+	slug: 'jetson-nano'
+	aliases: [ 'jetson-nano' ]
+	name: 'Nvidia Jetson Nano'
+	arch: 'aarch64'
+	state: 'experimental'
+
+	instructions: [
+		instructions.ETCHER_SD
+		BOARD_POWEROFF
+		BOARD_POWERON
+	]
+
+	gettingStartedLink:
+		windows: 'https://docs.resin.io/jetson-nano/nodejs/getting-started/#adding-your-first-device'
+		osx: 'https://docs.resin.io/jetson-nano/nodejs/getting-started/#adding-your-first-device'
+		linux: 'https://docs.resin.io/jetson-nano/nodejs/getting-started/#adding-your-first-device'
+
+	supportsBlink: false
+
+	yocto:
+		machine: 'jetson-nano'
+		image: 'resin-image'
+		fstype: 'resinos-img'
+		version: 'yocto-sumo'
+		deployArtifact: 'resin-image-jetson-nano.resinos-img'
```

Comment:
So we have the devkit with the SD card only? No internal storage?

Reply:
For the moment yes, release of emmc version is estimated to mid/end of June